cloud/ec2.go

```diff
@@ -183,6 +186,10 @@ func (m *ec2Manager) spawnOnDemandHost(ctx context.Context, h *host.Host, ec2Set
 				SubnetId:                 &ec2Settings.SubnetId,
 			},
 		}
+		if ec2Settings.IPv6 {
+			input.NetworkInterfaces[0].SetIpv6AddressCount(1)
```

Comment:
Is this a fluent interface? Can you `input.NetworkInterfaces[0].SetIpv6AddressCount(1).SetAssociatePublicIpAddress(false)`?

Reply:
We set input.NetworkInterfaces to a list with one item immediately before this, so it shouldn't be empty/nil. 
I probably can do that good catch!